src/OrchardCore.Modules/OrchardCore.AuditTrail/Views/AuditTrailEventDetail.DetailAdmin.cshtml

```diff
@@ -1,29 +1,31 @@
 @model AuditTrailEventViewModel
 
-<table class="table table-sm table-hover mb-0">
+<h3>@T["Event Info"]</h3>
```

Comment:
What's above this on the page is `h1`, so this should be an `h2`. Same for Content Item Info and User Info.

Reply:
it's makes the font way too large.

Reply:
I changed to to `<h2 class="h3"></h2>`

Reply:
That's what a Hungarian saying calls "iron hoop made of wood" :D.